src/components/grid/grid-item/grid-item.component.js

```diff
@@ -23,6 +23,8 @@ GridItem.propTypes = {
   children: PropTypes.node,
   /** How the grid item is aligned along the block (column) axis. Values: start, end, center, stretch */
   alignSelf: PropTypes.string,
+  /** Shorthand property for gridColumn and gridRow */
+  gridArea: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
We probably want to add this to the `responsiveSettings` prop types below as well I think

Reply:
I agree with Sam, I also had a little play around in a sandbox to refresh myself of how this component works and noticed the `gridArea` prop doesn't behave as I expect, the values do not override etc. 

I've set the same values on the element in the console to show what I'd expect to see what is currently seen in the styles

![image](https://user-images.githubusercontent.com/44157880/155974330-697567b3-eb38-469b-9444-90297f650b5a.png)


https://codesandbox.io/s/carbon-quickstart-forked-3xrlj8?file=/src/index.js

Reply:
that was caused by default values being set for `grid-column` and `grid-row`. I've replaced these defaults with `grid-area` now.